app/app/urls.py

```diff
@@ -66,6 +66,13 @@
     re_path(r'^health/', include('health_check.urls')),
     re_path(r'^lbcheck/?', retail.views.lbcheck, name='lbcheck'),
 
+    # grant views
+    path('grants/', include('grants.urls', namespace='grants')),
```

Comment:
Can we move this into `grants/urls.py` similarly to how `avatar/urls.py` is setup and referenced in this file?